HudiConnector/pom.xml

```diff
@@ -0,0 +1,309 @@
+<!--
```

Comment:
A readme would be useful explaining how to use this sample.

Reply:
I checked other examples none of other examples have README.md file. I have a blog explaining Hudi in depth.